### ISSUE TITLE
Window undefined

### ISSUE THREAD

**Issue by firdausious** [istarkov/google-map-react#228]
Hi,

I'm using server side rendering .. but somehow **_window** undefined.

```
 /Users/firdaus/myApps/node_modules/google-map-react/lib/utils/detectElementResize.js:29
     var raf = _window.requestAnimationFrame || _window.mozRequestAnimationFrame || _window.webkitRequestAnimationFrame || function (fn) {
                      ^

 TypeError: Cannot read property 'requestAnimationFrame' of undefined
     at /Users/firdaus/myApps/node_modules/google-map-react/lib/utils/detectElementResize.js:29:22
     at Object.<anonymous> (/Users/firdaus/myApps/node_modules/google-map-react/lib/utils/detectElementResize.js:35:4)
     at Module._compile (module.js:541:32)
     at Module._extensions..js (module.js:550:10)
     at require.extensions.(anonymous function) (/Users/firdaus/myApps/node_modules/babel-register/lib/node.js:154:7)
     at Object._module2.default._extensions.(anonymous function) [as .js] (/Users/firdaus/myApps/node_modules/require-hacker/babel-transpiled-modules/require hacker.js:260:68)
     at Module.load (module.js:458:32)
     at tryModuleLoad (module.js:417:12)
     at Function.Module._load (module.js:409:3)
     at Module.require (module.js:468:17)
```

How to solve that ? Thanks

**Comment by istarkov** [comment]
What version of this library?

**Comment by firdausious** [comment]
hi @istarkov 

react-google-maps@4.11.0

i'm using it on this boilerplate -> https://github.com/erikras/react-redux-universal-hot-example

**Comment by firdausious** [comment]
@istarkov oh.. it has been solved on 0.18.2 

all i need is just npm update.. ah great.. thanks..
